cli/flox/src/commands/containerize/macos_containerize_proxy.rs

```diff
@@ -70,6 +71,14 @@ impl ContainerBuilder for ContainerizeProxy {
             ]);
         }
 
+        // Honour FLOX_DISABLE_METRICS, if set.
+        if let Ok(disable_metrics) = std::env::var(FLOX_DISABLE_METRICS_VAR) {
+            command.args([
+                "--env",
+                &format!("{}={}", FLOX_DISABLE_METRICS_VAR, disable_metrics),
+            ]);
+        }
+
```

Comment:
maybe the value should indeed come from the config, as it is if you disable metrics through the system-wide config file in `/etc/`, they will be _enabled_ in the container.
alternatively we might want to mount the etc config file explicitly.

Reply:
looks like we set this env var earlier, maybe worth reminding us of that

Reply:
👍🏻 

>         // Honour `FLOX_DISABLE_METRICS` if set. Aside from being set by the
>         // user, it may also be set at runtime by  [Flox::Commands::FloxArgs]
>         // from another config path like `/etc/flox.toml` which isn't mounted
>         // into the proxy container.